sovtoken/sovtoken/request_handlers/write_request_handler/mint_handler.py

```diff
@@ -0,0 +1,51 @@
+from sovtoken.constants import MINT_PUBLIC, OUTPUTS
+from sovtoken.exceptions import UTXOError
+from sovtoken.messages.txn_validator import txn_mint_public_validate
+from sovtoken.request_handlers.token_utils import add_new_output
+from sovtoken.types import Output
+
+from indy_common.authorize.auth_actions import AuthActionAdd
+
+from plenum.common.exceptions import InvalidClientRequest, OperationError
+from plenum.common.request import Request
+from plenum.common.txn_util import get_seq_no, get_payload_data
+from plenum.server.database_manager import DatabaseManager
+from plenum.server.request_handlers.handler_interfaces.write_request_handler import WriteRequestHandler
+
+
+class MintHandler(WriteRequestHandler):
+
+    # it is not really clear what should be returned here for XFER
+    def gen_state_key(self, txn):
+        pass
+
+    def __init__(self, database_manager: DatabaseManager, txn_type, ledger_id, write_req_validator, state, utxo_cache):
```

Comment:
Also, state and utxo_cache should be registered into database_manager.
What is the txn_type? Is it MINT_PUBLIC?